lib/web_ui/lib/src/engine/view_embedder/dimensions_provider/custom_element_dimensions_provider.dart

```diff
@@ -45,11 +56,12 @@ class CustomElementDimensionsProvider extends DimensionsProvider {
 
   // Handle resize events
   late DomResizeObserver? _hostElementResizeObserver;
-  final StreamController<ui.Size> _onResizeStreamController =
-      StreamController<ui.Size>.broadcast();
+  late StreamSubscription<double>? _dprChangeStreamSubscription;
+  final StreamController<ui.Size?> _onResizeStreamController =
```

Comment:
Should we also change the type to `StreamController<void>` or `StreamController<Null>`?

Reply:
Nope, I really really want this to be a `StreamController<ui.Size>`, it's just that we don't know how to implement it efficiently *now* :P